styles/core.scss

```diff
@@ -137,3 +137,139 @@ SPDX-License-Identifier: Apache-2.0 */
     }
   }
 }
+
+.core-page-title {
+  background: var(--background-default);
+  position: sticky;
+  top: 0;
+  padding-top: 0.5rem;
+  margin-top: 4rem;
+  margin-bottom: 0.25rem;
+  padding-bottom: 0;
+  width: 100%;
+  z-index: 5;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: flex-end;
+  transition: padding 0.3s ease-out;
+
+  @media (max-width: v.$page-width-medium-threshold) {
+    top: 4rem;
+    padding-top: 0.75rem;
+    padding-bottom: 0.5rem;
+  }
+
+  .title {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 0;
+
+    > div {
+      &:last-child {
+        padding-left: 1rem;
+        flex-grow: 1;
+      }
+    }
+
+    button {
+      color: var(--text-color-secondary);
+      border: 1px solid var(--border-primary-color);
+      padding: 0.5rem 0.75rem;
+      margin: 0;
+      border-radius: 0.75rem;
+      font-size: 1.1rem;
+
+      &:hover {
+        background: var(--button-secondary-background);
+      }
+
+      .icon {
+        margin-left: 0.75rem;
+      }
+    }
+  }
+
+  h1 {
+    font-size: 1.75rem;
+    font-family: Unbounded, sans-serif;
+    position: relative;
+    transform: scale(1);
+    left: 0;
+
+    @media (max-width: v.$page-width-medium-threshold) {
+      left: -1rem;
+      transform: scale(0.75);
+    }
+
+    transition: all var(--transition-duration);
+    margin: 0;
+  }
+
+  .tabs {
```

Comment:
Everything in `tabs` should be in a different component, maybe `PageTitleTabs`, away from this one.